drud/ddev
# Run containers as UID 1000 on Windows

## The Problem/Issue/Bug:

OP #800 - Awkward warning (repeated lots) about Windows containers running as root.

There's really no reason to run them as root, we just couldn't work with Windows UID/GID. Go with uid 1000, which should work fine at least on Docker for Windows.

## Manual Testing Instructions:

Run on windows, verify correct behavior:
* Should be able to create upload files in sites/default/files
* Files created should be owned by the *host* user
* The warning should be gone.

## Automated Testing Overview:


## Related Issue Link(s):

## Release/Deployment notes:
